### PR TITLE
Update gitlab.md

### PR DESCRIPTION
**Pull request by jerry-li-dev**

### CODE REVIEW

File: src/pages/docs/integrations/available-integrations/gitlab.md [hunk 1]
```diff
@@ -19,91 +19,89 @@ Setting up a GitLab integration requires you to get a GitLab Personal Access Tok
 
 ## Generating a GitLab Personal Access Token
 
-Log in to [GitLab](https://about.gitlab.com/).
+Log in to [GitLab](https://gitlab.com/).
 
-If you don’t already have a Personal Access Token from GitLab, [generate a new one](https://gitlab.com/-/profile/personal_access_tokens).  
+If you don’t already have a Personal Access Token from GitLab, [generate a new one](https://gitlab.com/profile/personal_access_tokens).
```
Comment (jkonrath-postman) [inline src/pages/docs/integrations/available-integrations/gitlab.md:24]: Raj said this actually is https://gitlab.com/-/profile/personal_access_tokens with the /-/ in the URL. (PR #3126)

**Review by jkonrath-postman — CHANGES_REQUESTED** [review]
One change, but otherwise looks good!

File: src/pages/docs/integrations/available-integrations/gitlab.md (hunk 1 above)
Comment (jerry-li-dev) [reply src/pages/docs/integrations/available-integrations/gitlab.md]: Updated to new link

**Review by jkonrath-postman — APPROVED** [review]
Looks good to me!

**Pull request merged**